x/staking/types/delegation.go

```
@@ -8,7 +8,7 @@ import (
 	"cosmossdk.io/core/address"
```

golangci-lint (1.62.2)

File is not `gci`-ed with --skip-generated -s standard -s default -s prefix(cosmossdk.io) -s prefix(github.com/cosmos/cosmos-sdk) --custom-order

(gci)